client/src/main/webjar/navigation-list/navigation-list.html

```diff
@@ -13,6 +13,19 @@
         </div>
     </md-list-item>
 
+    <md-list-item ng-click="navigationList.viewPrograms()">
+        <md-icon md-font-library="material-icons">
+            pregnant_woman
```

Comment:
If we have a better icon for a program, I'm open to suggestions. We can also solicit feedback from customers tomorrow.